src/metabase/pulse/render/js_svg.clj

```diff
@@ -177,15 +177,6 @@
                                                   (json/generate-string settings)))]
     (svg-string->bytes svg-string)))
 
-(defn categorical-bar
```

Comment:
This function isn't used.

Reply:
Ah, good catch! My local dev. says that 'categorical-area' isn't used either.